src/Go/ui/GoHistoryPage.tsx

```diff
@@ -115,15 +115,20 @@ export const GoHistoryPage = (): React.ReactElement => {
                   <Tooltip
                     title={
                       <>
-                        Two wins in a row against a faction will give you +1 favor to that faction <br />
-                        (up to a max of {getMaxFavor()} favor), if you are a member of that faction <br />
+                        Two wins in a row against an opponent will give you 1000 rep converted to favor with that
+                        faction <br />
+                        (up to a max of {getMaxRep()} favor), if you are a member of that faction.
+                        <br />
+                        <br />
+                        This is done as favor converted to rep to avoid making the favor worth much less rep early in an
```

Comment:
I would remove this, it is confusing and irrelevant. Players don't care about why we made a change. Instead, explain the consequences of having rep immediately converted to favor: 
"The rep is immediately applied as favor, which means it is useful for certain purposes right away, without needing to install augs."